### ISSUE TITLE
XOR a, a Flags

### ISSUE THREAD

**Issue by the-pink-hacker** [deeptoaster/opcode-table#8]
The opcode `AF` also known as instruction `xor a, a` guarantees that a is 0. The zero flag is labeled as "as defined". This is in fact true, but since the flag will always be set no matter what, shouldn't it say set instead?

Sorry if this is too nit-picky.

**Comment by dt-anchorzero** [comment]
That's true, and this also applies to a few more instructions (`cp a`, `sub a`, etc.).

The reason it reads this way is because `xor 𝑟` is generated in common for all seven registers, but I'll take a look and see if we can add it as a special case. Thanks for the suggestion!
